[draft] MD-118: Implications of L1 assumptions for postconfirmation protocol

apenzk opened the pull request:
# Summary

[MD-118](https://github.com/movementlabsxyz/MIP/blob/md/L1-and-postconfirmation/MD/md-118/README.md)

apenzk committed:
init

apenzk committed:
minor edits

apenzk committed:
md number

apenzk committed:
improve header

apenzk committed:
minor edits

apenzk committed:
appendix A1 - message propagation

l-monninger commented:
I think this should ask for a list of...
- Safety assumptions (which components are safe) and to what extent.
- Liveness assumptions (which components are live) and to what extent.
- Synchrony assumptions (which components are synchronous) and to what extent. 
- Finality assumptions--when does finality occur and what exactly does it mean.

l-monninger commented:
Maybe #110 is a useful super framework to improve here?

l-monninger commented:
What is the weakest set of assumptions under which a given protocol works?

apenzk committed:
make desiderata more concrete

apenzk committed:
address review

apenzk commented:
@l-monninger 
> Finality assumptions--when does finality occur and what exactly does it mean.

individually address this now as D3

> Safety assumptions (which components are safe) and to what extent.
> Liveness assumptions (which components are live) and to what extent.
> Synchrony assumptions (which components are synchronous) and to what extent.

added.

> What is the weakest set of assumptions under which a given protocol works?

added.
